### PR TITLE
doc fix for KLDivLoss

### PR DESCRIPTION
**Pull request by nikitaved**
Fixes doc for KLDivLoss as per [this comment](https://github.com/pytorch/pytorch/pull/34586#discussion_r404442741).

### CODE REVIEW

File: torch/nn/modules/loss.py [hunk 1]
```diff
@@ -294,7 +294,7 @@ class KLDivLoss(_Loss):
 
     As with :class:`~torch.nn.NLLLoss`, the `input` given is expected to contain
     *log-probabilities* and is not restricted to a 2D Tensor.
-    The targets are given as *probabilities* by default, but could be passed
+    The targets are interpretet as *probabilities* by default, but could be considered
```
Comment (rgommers) [inline torch/nn/modules/loss.py:297]: typo: `interpretet` -> `interpreted`

Comment (nikitaved) [reply torch/nn/modules/loss.py]: What an epic failure :) Than you!

**Review by gchanan — APPROVED** [review]
thanks!
